mkosi

```diff
@@ -429,7 +433,7 @@ def install_debian_or_ubuntu(args, workspace, run_build_script, mirror):
     cmdline = ["debootstrap",
                "--verbose",
                "--variant=minbase",
-               "--include=systemd",
+               "--include=systemd,usrmerge",
```

Comment:
@stefan-it hmm. You are correct. We also need to detect the release to create bootable images, as kernel-install is not available in jessie.